.env.example

```
@@ -44,3 +103,29 @@ SENDGRID_DATA_RESIDENCY=
 # Legacy fallback key supported in code:
 SENDGRID_LIST_ID=
 SENDGRID_MAILING_ID=
+
+# GitHub-powered tech curation cron.
+GITHUB_TOKEN=
+GITHUB_OWNER=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GITHUB_OWNER key should go before the GITHUB_TOKEN key

(UnorderedKey)